specification/cognitiveservices/data-plane/Personalizer/preview/v2022-09-01-preview/Personalizer.json

```diff
@@ -0,0 +1,2847 @@
+						"required": true,
+						"type": "string",
+						"format": "date-time"
+					},
+					{
+						"in": "query",
+						"name": "window",
+						"description": "Rolling or Expanding time. Rolling compatible with 1 hour, 6 hour, 12 hour and 1 day time intervals. Expanding compatible with 5 minute time interval only. Defaults to Rolling.",
+						"type": "string",
+						"enum": [
+							"Rolling",
+							"Expanding"
+						],
+						"x-ms-enum": {
+							"name": "Window",
+							"modelAsString": true
+						}
+					},
+					{
+						"in": "query",
+						"name": "filter",
+						"description": "An expression to filter result array.",
+						"type": "string"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "Success",
+						"schema": {
+							"$ref": "#/definitions/Evaluation"
+						}
+					},
+					"default": {
+						"description": "Offline Evaluation not found.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					}
+				},
+				"x-ms-examples": {
+					"Successful Evaluations_Get request": {
+						"$ref": "./examples/Evaluations_Get.json"
+					}
+				}
+			},
+			"delete": {
+				"tags": [
+					"Evaluations20220901Preview"
+				],
+				"summary": "Delete Evaluation.",
+				"description": "Delete the Offline Evaluation associated with the Id.",
+				"operationId": "Evaluations_Delete",
+				"consumes": [],
+				"produces": [],
+				"parameters": [
+					{
+						"name": "api-version",
+						"in": "query",
+						"required": true,
+						"type": "string",
+						"description": "Client API version."
+					},
+					{
+						"name": "evaluationId",
+						"in": "path",
+						"description": "Id of the Offline Evaluation to delete.",
+						"required": true,
+						"type": "string",
+						"maxLength": 256
+					}
+				],
+				"responses": {
+					"204": {
+						"description": "Success"
+					}
+				},
+				"x-ms-examples": {
+					"Successful Evaluations_Delete request": {
+						"$ref": "./examples/Evaluations_Delete.json"
+					}
+				}
+			}
+		},
+		"/evaluations": {
+			"get": {
+				"tags": [
+					"Evaluations20220901Preview"
+				],
+				"summary": "List Offline Evaluations.",
+				"description": "List of all Offline Evaluations.",
+				"operationId": "Evaluations_List",
+				"consumes": [],
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"name": "api-version",
+						"in": "query",
+						"required": true,
+						"type": "string",
+						"description": "Client API version."
+					},
+					{
+						"in": "query",
+						"name": "filter",
+						"description": "An expression to filter the evaluations against evaluations metadata. Only evaluations where the expression evaluates to true are included in the response.",
+						"type": "string"
+					},
+					{
+						"in": "query",
+						"name": "orderby",
+						"description": "The ordering. Defaults to creation time in descending order.",
+						"type": "string",
```

Comment:
orderby parameter must be `type: array` with items of `type: string`

Reply:
During api review board, we decided to remove the order property.

Reply:
Deleted property.